config/features/config.go

```diff
@@ -282,11 +284,29 @@ func ConfigureBeaconChain(ctx *cli.Context) error {
 		cfg.ForceHead = ctx.String(forceHeadFlag.Name)
 	}
 
+	if ctx.IsSet(blackListRoots.Name) {
+		logEnabled(blackListRoots)
+		cfg.BlacklistedRoots = parseBlacklistedRoots(ctx.StringSlice(blackListRoots.Name))
+	}
+
 	cfg.AggregateIntervals = [3]time.Duration{aggregateFirstInterval.Value, aggregateSecondInterval.Value, aggregateThirdInterval.Value}
 	Init(cfg)
 	return nil
 }
 
+func parseBlacklistedRoots(blacklistedRoots []string) map[[32]byte]struct{} {
+	roots := make(map[[32]byte]struct{})
+	for _, root := range blacklistedRoots {
+		r, err := bytesutil.DecodeHexWithLength(root, 32)
+		if err != nil {
+			log.WithError(err).WithField("root", root).Warn("Failed to parse blacklisted root")
```

Comment:
Thinking about this more - IMO we really want to raise an error here. The user tried to specify an invalid block - which is an exceptional circumstance. Imagine using this in the holesky scenario - you mis-paste the block, don't see this log, and all the sudden your node is back on the bad chain. The function calling this one returns an error so it would be easy to plumb that through. It would also simplify the tests on this method, checking the error instead of dealing with log tests.

Reply:
raising an error and stopping the node?